apps/feed/client/index.coffee

```diff
@@ -20,8 +21,7 @@ module.exports = ->
     }, $homeComponent)
 
   if ($emptyComponent = $('.js-empty-feed')).length
-    showModal = qs.parse(location.search.replace('?', ''))?.showModal
```

Comment:
Nah, I did it in the component.

Reply:
This had to be changed because you get a warning for calling a function like that inside of a render call. So switching to the portal style modal causes a mismatch on client/server. So this happens after mounting.

Reply:
Ah got it